tests/test_conversions.py

```diff
@@ -0,0 +1,42 @@
+import pytest
+import pandas as pd
+import numpy as np
+
+from jhive_previz import conversions as conv
+from jhive_previz import dataproc
+
+
+@pytest.mark.parametrize(
+    "input_unit,output_unit,expected",
+    [
+        ("microJansky", "magnitude", conv.flux_to_mag),
+        pytest.param("Jansky", "magnitude", conv.flux_to_mag, marks=pytest.mark.xfail),
+    ],
+)
+def test_get_conversion_function(input_unit, output_unit, expected):
+    """Make sure that get_conversion_function works as expected"""
```

Comment:
Test docstrings: describe what's actually being tested (i.e., what is the true condition of the test)

Reply:
Done!